WcaOnRails/spec/helpers/application_helper_spec.rb

```diff
@@ -32,7 +32,14 @@
     it "handles multiple phrases correctly highlighting them" do
       text = "A long text with a super word in the middle (directly here) followed by the rest of an awesome and peculiar sentence. What do you think?"
       expected = "...t with a super word in the middle (directly here) <strong>followed</strong> by the rest of an awesome and peculiar sentence. What do you <strong>think</strong>?"
-      expect(helper.wca_excerpt(text, ["followed", "think"])).to eq expected
+      expect(helper.wca_excerpt(text, %w(followed think))).to eq expected
+    end
+
+    it "is case insensitive" do
```

Comment:
I hear you, anyway I really prefer the 'small, simple tests' approach. If we'd think about runtime and about less code (sometimes we can easily test multiple things at once), we could end up with long tests, and the idea behind them is hard to figure out. IMO ideally each test has good description and the code does exactly what the description says. For example when `it "works"` fails we usually need to look into the code anyway.
Especially, this test is not that obvious even now as it test kinda tricky thing.